### PR TITLE
[tflite] make label_image build

### PR DESCRIPTION
**Pull request by freedomtan**
label_image depends on old `//tensorflow/lite/delegates/gpu:gl_delegate`, which no long built.
update the dependency to use `//tensorflow/lite/delegates/gpu:delegate`

### CODE REVIEW

**Review by impjdi — APPROVED** [review]
Thanks! :)

File: tensorflow/lite/examples/label_image/BUILD [hunk 1]
```diff
@@ -39,11 +39,11 @@ cc_binary(
         "@com_google_absl//absl/memory",
     ] + select({
         "//tensorflow:android": [
-            "//tensorflow/lite/delegates/gpu:gl_delegate",
+            "//tensorflow/lite/delegates/gpu:delegate",
```
Comment (impjdi) [inline tensorflow/lite/examples/label_image/BUILD:42]: can you actually also add #include ".../delegate.h"?  it's being currently pulled in indirectly, but we should have explicit #include.

**Review by impjdi — CHANGES_REQUESTED** [review]

File: tensorflow/lite/examples/label_image/BUILD (hunk 1 above)
Comment (freedomtan) [reply tensorflow/lite/examples/label_image/BUILD:42]: OK. Thanks

**Review by impjdi — DISMISSED** [review]

File: tensorflow/lite/examples/label_image/label_image.cc [hunk 2]
```diff
@@ -45,6 +45,13 @@ limitations under the License.
 #include "tensorflow/lite/string_util.h"
 #include "tensorflow/lite/tools/evaluation/utils.h"
 
+#if defined(__ANDROID__)
+#include "tensorflow/lite/delegates/gpu/delegate.h"
+#if (defined(__arm__) || defined(__aarch64__))
+#include "tensorflow/lite/experimental/delegates/hexagon/hexagon_delegate.h"
```
Comment (terryheo) [inline tensorflow/lite/examples/label_image/label_image.cc:51]: Is this needed? CreateHexagonDelegate() is defined in lite/tools/evaluation/utils.h

Comment (freedomtan) [reply tensorflow/lite/examples/label_image/label_image.cc:51]: @terryheo nope, they are not necessary. Both GPU and Hexagon headers are included in `lite/tools/evaluation/utils`. Changed per comment of @impjdi.

Comment (terryheo) [reply tensorflow/lite/examples/label_image/label_image.cc:51]: gpu/delegate.h is needed for some definition such as TFLITE_GPU_INFERENCE_PREFERENCE_SUSTAINED_SPEED. But I don't think we need to include hexagon_delegate.h here if it's not needed.

Comment (freedomtan) [reply tensorflow/lite/examples/label_image/label_image.cc:51]: Thanks explanation. I'll remove it. BTW, how can I verify if some headers are needed?

**Review by terryheo — APPROVED** [review]
LGTM

**Pull request merged**